src/Microsoft.DotNet.VersionTools/Util/ExponentialRetry.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This seems like it would be a more generally useful task....is there a better place to put this?

Reply:
Hmm, yeah, I'll put it in `src/common`. VersionTools is imported by the main BuildTools task proj and the Feeds task proj, but maybe CloudTestTasks or something else would find it useful too.